Frameworks/jdi-core/src/main/java/com/epam/jdi/uitests/core/initialization/CascadeInit.java

```diff
@@ -86,16 +131,50 @@ private IBaseElement getInstancePage(Object parent, Field field, Class<?> type,
         return instance;
     }
 
+    /**
+     * Get instance element
+     * @param parent
+     * @param type
+     * @param parentType
+     * @param field
+     * @param driverName
+     * @return IBaseElement
+     */
     private IBaseElement getInstanceElement(Object parent, Class<?> type, Class<?> parentType, Field field, String driverName) {
         return createChildFromFieldStatic(parent, parentType, field, type, driverName);
     }
 
+    /**
```

Comment:
Defined method arguments with no description are not informative at all
 <img width="738" alt="empty arguments description" src="https://user-images.githubusercontent.com/12053985/45762751-c5a5ca00-bc37-11e8-8ee6-c8d1e963fae6.PNG">